app/assets/javascripts/nfg_ui/application.coffee

```diff
@@ -3,16 +3,14 @@
 #= require popper
 #= require select2
 #= require bootstrap
-#= require moment
-#= require nfg_ui/vendor/bootstrap-datetimepicker.min
 #= require nfg_ui/vendor/legacy_browser_support/application
 
 #= require_self
 #= require nfg_ui/document_ready
 #= require nfg_ui/collapsible_toggle
 #= require nfg_ui/select2
 #= require nfg_ui/tooltips
-#= require nfg_ui/datetimepicker_configuration
 #= require nfg_ui/file_inputs
+#= require nfg_ui/datepicker
```

Comment:
just refactoring so we don't have to include entire application.coffee when only datepicker is required. 
Also bootstrap versions clash with each other when host projects require nfg ui's application.coffee.